homeassistant/components/snapcast/__init__.py

```diff
@@ -11,15 +11,13 @@
 
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up Snapcast from a config entry."""
-    host = entry.data[CONF_HOST]
-    port = entry.data[CONF_PORT]
-    coordinator = SnapcastUpdateCoordinator(hass, host, port)
+    coordinator = SnapcastUpdateCoordinator(hass, entry)
 
     try:
         await coordinator.async_config_entry_first_refresh()
     except OSError as ex:
         raise ConfigEntryNotReady(
-            f"Could not connect to Snapcast server at {host}:{port}"
+            f"Could not connect to Snapcast server at {entry.data[CONF_HOST]}:{entry.data[CONF_PORT]}"
```

Comment:
We should break up this string (or keep the host and port assignments and revert the changes)

Reply:
I think break up is ok :+1: